package.json

```diff
@@ -91,17 +91,17 @@
     "eslint-plugin-simple-import-sort": "^12.1.1",
     "eslint-plugin-tsdoc": "^0.4.0",
     "eslint-plugin-unused-imports": "^4.1.4",
-    "express": "^4.21.2",
+    "express": "^5.1.0",
     "gcp-metadata": "^5.3.0",
     "js-yaml": "^4.1.0",
-    "mocha": "^10.8.2",
+    "mocha": "^11.1.0",
     "mocha-sinon": "^2.1.2",
     "mongodb-client-encryption": "^6.3.0",
     "mongodb-legacy": "^6.1.3",
-    "nyc": "^15.1.0",
-    "prettier": "^3.4.2",
+    "nyc": "^17.1.0",
```

Comment:
Seems like something's wrong with the coverage task, is the major adoptable?

Reply:
I don't think this is related.  Are we seeing this elsewhere?

```
[2025/04/29 18:17:42.080] fatal error: An error occurred (AccessDenied) when calling the ListObjectsV2 operation: User: arn:aws:sts::557821124784:assumed-role/evergreen_task_hosts_instance_role_production/i-0c7738ff311257e60 is not authorized to perform: s3:ListBucket on resource: "arn:aws:s3:::mciuploads" because no identity-based policy allows the s3:ListBucket action
```

Reply:
This is on main:

https://parsley.mongodb.com/evergreen/mongo_node_driver_next_generate_combined_coverage_download_and_merge_coverage_de2c9552a961f1189a7e7fdd294e688c6bbec031_25_04_28_19_03_53/1/task?bookmarks=0,1109&shareLine=1042

```
sed: couldn't edit coverage: not a regular file
```

So something funky but that means we haven't actually checked if this version works if it's not passing right?

Reply:
I did think it was passing.

I tried investigating this issue but CI has been really flaky for me yesterday/this morning.  I also ran the coverage reports locally and saw no difference in the output files generated between versions 🤔.  I might just revert this for now and hope next quarter's dependency updates have a more stable CI for testing.